Scroller/Scroller.js

```diff
@@ -32,6 +32,7 @@ import ScrollbarPlaceholder from '../useScroll/ScrollbarPlaceholder';
 import Skinnable from '../Skinnable';
 
 import {EditableShape, EditableWrapper} from './EditableWrapper';
+import {ContentContainerDecorator} from './ContentContainerDecorator';
```

Comment:
Let's move this line up for alphabetical order of imported module names.

Reply:
It's moved